pkg/templateservicebroker/servicebroker/provision.go

```diff
@@ -85,7 +86,12 @@ func (b *Broker) ensureTemplateInstance(u user.Info, namespace string, instanceI
 	}
 
 	templateInstance := &templateapi.TemplateInstance{
-		ObjectMeta: metav1.ObjectMeta{Name: instanceID},
+		ObjectMeta: metav1.ObjectMeta{
+			Name: instanceID,
+			Annotations: map[string]string{
+				oapi.OpenShiftExternalServiceID: instanceID,
```

Comment:
Is this the same as the `service_id` from the provision request?

https://github.com/openservicebrokerapi/servicebroker/blob/master/spec.md#provisioning

I believe we could then tie that back to the k8s service instance in the console, since the k8s ServiceInstance object has a `spec.externalID`

https://github.com/kubernetes-incubator/service-catalog/blob/master/pkg/apis/servicecatalog/v1alpha1/types.go#L430-L433

Reply:
No, it's the instance_id, but yes, it should be identical to the spec.externalID in the ServiceInstance.
Hold on, I'm going to do an s/ExternalService/Instance/ on this PR.

Reply:
Perfect. Yes, that's what we need.